test/dshell/extra-files/dwarf/fix22468.d

```diff
@@ -0,0 +1,5 @@
+/*
+EXTRA_ARGS: -g -defaultlib= -betterC -c
```

Comment:
I'm compiling with those flags to make the compiler not generate other types other than `dchar`.